src/Workspaces/Core/Portable/Remote/IRemoteHostService.cs

```diff
@@ -13,7 +13,6 @@ internal interface IRemoteHostService
         Task SynchronizeGlobalAssetsAsync(Checksum[] checksums, CancellationToken cancellationToken);
 
         void RegisterPrimarySolutionId(SolutionId solutionId, string storageLocation, CancellationToken cancellationToken);
-        void UnregisterPrimarySolutionId(SolutionId solutionId, bool synchronousShutdown, CancellationToken cancellationToken);
```

Comment:
so this is basically send out to let it know current working folder.

Reply:
Right... but why did we get rid of Unregister.  Is that not needed anymore by the remote workspace?